### ISSUE TITLE
"SS2OSD.dll" caused an Access Violation

### ISSUE THREAD

**Issue by nhillen** [Hotrian/OVRdrop-Public#56]
Im getting this 100% of the time trying to run OVRdrop

"Unity Player [version: Unity 5.3.6f1_29055738eb78]

SS2OSD.dll caused an Access Violation (0xc0000005)
  in module SS2OSD.dll at 0023:64d0439b."

[error.log](https://github.com/Hotrian/OVRdrop-Public/files/2363997/error.log)
[output_log.txt](https://github.com/Hotrian/OVRdrop-Public/files/2363998/output_log.txt)

**Comment by Hotrian** [comment]
SS2OSD is doing something weird to the DirectX stack. Please close Sonic Studio 2 before running OVRdrop.

**Comment by Eincrou** [comment]
Yep, I also have an ASUS motherboard, and I have had to manually close the Sonic Studio process entirely, even though I don't have the Sonic Radar on screen display enabled.
